pages/home/addTask.js

```diff
@@ -39,12 +42,16 @@ export default function AddTask() {
   async function handleSubmit(event) {
     event.preventDefault();
     await submitTask();
-    setPopup(!popup);
+    triggerPopup();
```

Comment:
You could just write:

```suggestion
   setPopup(previousState => !previousState)
```

Reply:
thanks for your suggestion, i fixed it 👍